README.md

```diff
@@ -0,0 +1,15 @@
+# Mautic developer documentation (new)
+
+This repository hosts the new developer documentation for Mautic on the [Read the Docs platform](https://readthedocs.org/). Whenever a PR is merged, changes are deployed immediately to https://mautic-developer.readthedocs.io/
```

Comment:
Can we move links to [macros](https://docs.mautic.org/en/contributing/style-guide#absolute-links), assuming RTD supports it?

Reply:
Done in https://github.com/mautic/developer-documentation-new/pull/3/commits/1e9b2848c030650af3b457ae6f62cf49d964d2d2. Thanks for the feedback 😊